dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -297,6 +298,58 @@ public Organization updateOrganizationDescription(@ApiParam(hidden = true) @Auth
         return eventDAO.findEventsForOrganization(id, offset, limit);
     }
 
+    @PUT
+    @Timed
+    @UnitOfWork
+    @Path("/{organizationId}/star")
+    @ApiOperation(value = "Star an organization.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) })
+    public void starOrganization(@ApiParam(hidden = true) @Auth User user,
+                          @ApiParam(value = "Organization to star.", required = true) @PathParam("organizationId") Long organizationId,
+                          @ApiParam(value = "StarRequest to star an organization for a user", required = true) StarRequest request) {
+        Organization organization = organizationDAO.findApprovedById(organizationId);
+        boolean isOrgApproved = isOrganizationApproved(organizationId);
```

Comment:
You already have the organization from line 309. Your `isOrganizationApproved` is potentially doing another DB call, which is expensive (potentially because Hibernate may cache it). You should get rid of this line and do something like

```
throwErrorIfOrganizationNotFound(organization != null)
```

Also just noticed, the more common pattern we have is `check...`, so I'd consider renaming throwErrorIfOrganizationNotFound to `checkOrganization`, but not crucial.